### PR TITLE
Align flow inputs and outputs names to snake case

### PR DESCRIPTION
**Pull request by mihaitusa**
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
closes #407

**Comment by mihaitusa** [comment]
Should not be merged until further tests in sections without automation.

**Comment by mihaitusa** [comment]
@gituser173  Please review it! Not to be merged yet.

File: content/io/cloudslang/base/network/rest/http_client_action.sl [hunk 1]
```diff
@@ -72,108 +107,224 @@ operation:
...
+    - http_client_cookie_session:
+        required: false
     - httpClientCookieSession:
+        default: ${get("http_client_cookie_session", "")}
+        overridable: false
+    - http_client_pooling_connection_manager:
         required: false
     - httpClientPoolingConnectionManager:
-        required: false
-    - validHttpStatusCodes:
```
Comment (gituser173) [inline content/io/cloudslang/base/network/rest/http_client_action.sl]: is validHttpStatusCodes an input of the java action?

File: content/io/cloudslang/base/os/linux/users/add_user_to_sudoers_list.sl [hunk 2]
```diff
@@ -8,18 +8,29 @@
 ####################################################
 # This flow performs a linux command to add a specified user to sudoers group
 #
-#    Inputs:
-#      - host - hostname or IP address
-#      - port - optional - port number for running the command
-#      - password - optional - password of user
-#      - private_key_file - optional - the path to the private key file
-#      - user: - the user to be added in sudoers group
```
Comment (gituser173) [inline content/io/cloudslang/base/os/linux/users/add_user_to_sudoers_list.sl]: delete ":" after user

**Comment by gituser173** [comment]
Looks good :+1:

File: content/io/cloudslang/base/network/rest/http_client_action.sl (hunk 1 above)
Comment (mihaitusa) [inline content/io/cloudslang/base/network/rest/http_client_action.sl]: no... is just an input for http_client_action operation used for testing statusCode outputs.

**Pull request merged**